### PR TITLE
[8.0][IMP] mrp_operations_extesion: fix warning in log

### PR DESCRIPTION
**Pull request by oihane**

### CODE REVIEW

File: mrp_operations_extension/models/mrp_bom.py [hunk 1]
```diff
@@ -13,8 +13,8 @@ class MrpBom(models.Model):
     def _prepare_wc_line(self, wc_use, level=0, factor=1):
         res = super(MrpBom, self)._prepare_wc_line(
             wc_use, level=level, factor=factor)
-        if not self.env['mrp.config.settings']._get_parameter('cycle.by.bom',
-                                                              False) == 'True':
+        if not self.env['mrp.config.settings']._get_parameter(
+                'cycle.by.bom').value:
```
Comment (pedrobaeza) [inline mrp_operations_extension/models/mrp_bom.py]: What happens if the parameter doesn't exist?

Comment (oihane) [reply mrp_operations_extension/models/mrp_bom.py]: If the parameter exists it will have value = 'True', if it shall be False, parameter won't exist.

Comment (pedrobaeza) [reply mrp_operations_extension/models/mrp_bom.py]: But then `False.value` will throw an error...

**Comment by Daniel-CA** [comment]
:+1:

**Review by pedrobaeza — APPROVED** [review]

**Comment by esthermm** [comment]
:+1:

**Comment by oihane** [comment]
I've taken a look to the tests in branch 8.0 it takes [6.399s](https://travis-ci.org/OCA/manufacture/jobs/227919465#L1233) to pass all the test in this module, and in this PR [4.961s](https://travis-ci.org/OCA/manufacture/jobs/230294285#L1225)
it is not a big difference but it counts.

cc @jbeficent @tafaRU @andhit-r

**Review by tafaRU — APPROVED** [review]

**Comment by anajuaristi** [comment]
:+1:

**Pull request merged**